src/Sdk/Models/AuditLog.cs

```diff
@@ -1,9 +0,0 @@
-﻿namespace Passwordless.Net;
-
-public class AuditLog

```

Comment:
@jonashendrickx When did we add this to the SDK?

Reply:
This was from us/you haha, we might need to add a class like it in the future but we should remove it for now.